src/Utility.cs

```
@@ -48,9 +27,9 @@ internal static string ResolveConnectionString(string attributeConnectionStringK
                     return resolvedString;
                 }
             }
-            catch (Exception e)
-            {

```

nit: Undo and please add a comment `//ignore exception`